spec/unit/peep_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'peep'
+require 'database_helpers'
+
+describe Peep do
+
+    describe '.all' do
+        it 'returns a list of peeps' do
+            Peep.post(message: "Test peep 1")
+            Peep.post(message: "Test peep 2")
+            Peep.post(message: "Test peep 3")
+            peeps = Peep.all
+
+            expect(peeps.length).to eq 3
+            expect(peeps.first).to be_a Peep
+        end
+    end
```

Comment:
Not sure you have a test for reverse chronological order. Could just change bottom of .all test to be expect(peeps.first).to eq('Test peep 3')